### ISSUE TITLE
It doesn't seem to decode

### ISSUE THREAD

**Issue by r3v2d0g** [Krognol/go-pure#11]
Hi!

I'm trying to use your lib but it doesn't seem to decode my file. So here is what I tried:

```go
package main

import (
	"fmt"
	"io/ioutil"
	"os"

	"github.com/Krognol/go-pure"
)

type ConfigPure struct {
	Token string `pure:"token"`
}

func main() {
	Token := "somerandomishtoken"

	ce := &ConfigPure{Token}
	b, err := pure.Marhsal(ce)
	if err != nil {
		panic(err)
	}
	f, _ := os.Create("encode_test.pure")
	_, nerr := f.Write(b)
	if nerr != nil {
		panic(nerr)
	}

	cd := &ConfigPure{}
	b, _ = ioutil.ReadFile("encode_test.pure")
	err = pure.Unmarshal(b, cd)
	if err != nil {
		println(err.Error())
		os.Exit(1)
	}

	fmt.Printf("%#v\n", cd)
}
```

But it returns `&main.ConfigPure{Token:""}` even though the content of `encode_test.pure` is:

```
token = "somerandomishtoken"

```

Thank you for your help =)

**Comment by Krognol** [comment]
It seems to work on my end. Can you try updating the library? `go get -u github.com/Krognol/go-pure/pure`
And then change the import to `github.com/Krognol/go-pure/pure`

**Comment by r3v2d0g** [comment]
Hey!

First of all, thanks for the response.
So, in fact I also tried with the right `import` path before but I just wrote it wrong when I created this issue.

In fact, the issue was that I was stupid enough to think that I could use `glide` and a `vendor` directory outside the `$GOPATH` (so, of course when I was importing `go-pure` it was importing a very old version and not the updated one that was in the directory).

Problem fixed 😄 

P.S: you are the best for writing this library and I really hope that it will get the attention it deserves ❤️
